oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -114,9 +114,8 @@ References
       links:
         - url: "https://docs.openedx.org"
           title: "Deployed Site"
-          # Backstage uses the MaterialUI Icon Set.
-          # https://mui.com/material-ui/material-icons/
-          # The value will be the name of the icon.
+          # Backstage uses the MaterialUI Icon Set (https://mui.com/material-ui/material-icons/)
+          # The value will be the name of the icon. We recommend using "Web" for all icons.
```

Comment:
If we want people to have the flexibility of choosing something more appropriate for certain repos, we could make this small change.
```suggestion
          # The value will be the name of the icon. We recommend using "Web".
```